modules/retail/pages/retail-formula-bind.adoc

```diff
@@ -21,12 +21,13 @@ Zone 2 is the reverse zone of Zone 1.
 Its main purpose is to resolve IP addresses back to hostnames.
 Zone 2 is primarily needed for the correct determination of the FQDNs of the branch.
 
-
+// REMARK: this procedure is probably too long.  Where to split?
+// REMARK: are these list items (`*`) substeps?  Or what?  Confusing.
```

Comment:
They are substeps: they're basically all the fields on a single dialog. I struggled to put this procedure together, as you can tell! I think I originally had the reverse settings as a separate procedure, but they've merged together again at some point.

Reply:
yes ;)  When I continued reviewing I saw that it is consistent and thus it makes sense.